xla/service/gpu/transforms/cudnn_fused_conv_rewriter.cc

```diff
@@ -576,33 +707,81 @@ void CaptureConvGraphRecursive(HloInstruction* instr,
       std::optional<PrimitiveType> f8_type = IsSaturatingCastToF8(users_user);
```

Comment:
I think you should always check there is just one user of `user`. Otherwise you might miss an endpoint.